src/template-cache.js

```diff
@@ -77,7 +78,7 @@ _.extend(TemplateCache.prototype, DomMixin, {
   // using a template-loader plugin as described here:
   // https://github.com/marionettejs/backbone.marionette/wiki/Using-marionette-with-requirejs
   loadTemplate(templateId, options) {
-    const $template = this.findEls(templateId);
+    const $template = Backbone.$(templateId);
```

Comment:
So, as I understand it, we're constraining the DOM API to just operate on views - and removing it from `CompositeView` due to it being deprecated.

Do you see the `TemplateCache` implementation being removed in future or is it just generally something that people who aren't using jQuery tend not to need?